src/index.js

```diff
@@ -3,22 +3,39 @@
  */
 
 // example of generating HTML with js
-function renderSidebar() {
-  const sidebarContents = `
-    <h2 class="email-header">Inbox</h2>
-    <ul class="email-list">
-      <li>
-        <button class="email-item" type="button">
-          <div class="sender-details">
-            <p>Email sender</p>
-            <span>Timestamp</span>
-          </div>
-          <p class="email-subject">Email1 subject</p>
-          <p>Email snippet</p>
-        </button>
-      </li>
-    </ul>
-  `;
+import store, {getMessageSubject, getMessageSender, formatSender, formatTimestamp} from './store'
+
+function renderThreads() {
+  const { threadIds } = store.mailboxes.INBOX;
+  const { messages } = store;
+
+  if (store.mailboxes.INBOX == null) return;
+
+  return threadIds.map( ind => {
+    const subject = getMessageSubject(messages[ind]);
+    const snippet = messages[ind].snippet;
+    const sender = getMessageSender(messages[ind]);
+    const timestamp = formatTimestamp(messages[ind].internalDate);
```

Comment:
Teeny little thing but I’d DRY this up by making a temporary variable for `messages[ind]`